dask/tests/test_base.py

```diff
@@ -405,6 +405,22 @@ def test_tokenize_ordered_dict():
     assert tokenize(a) != tokenize(c)
 
 
+try:
+    import dataclasses
+
+    # Avoid @dataclass decorator as Python < 3.7 fail to interpret the type hints
+    ADataClass = dataclasses.make_dataclass("ADataClass", [("a", int)])
+except ImportError:
+    dataclasses = None
```

Comment:
I know this was just moved but the try/except can be dropped now. dataclasses is a builtin module in py3.7+ and we do no longer support py3.6

```suggestion
from dataclasses import dataclass

@dataclass
class ADataClass:
    a: int
```